pipeline/transform/user_interests.py

```diff
@@ -3,16 +3,37 @@
 import prefect
 from prefect import task
 
-from pipeline.types import RawUserInterests, User
+from pipeline.types import Interest, RawUserInterests, User
 
 
 @task
 def augment_users_with_interests(
     users: List[User], raw_interests: RawUserInterests
 ) -> List[User]:
-    # TODO: Implement your transform task
 
     logger = prefect.context.get("logger")
-    logger.info("Chris")
+    logger.info(
+        "Finished issues #103 & #194 (Chris); wrote additional test cases as well"
+    )
+
+    # TODO: Implement your transform task
+
+    for user in users:
+
+        interest_lst = []
+
+        try:
+
+            for key, value in raw_interests[user.uid].items():
```

Comment:
Optional advice: Instead of using a `try/except` for `KeyError` you can also handle a possibly missing key like this:

```python
for key, value in raw_interests.get(user.uid, {}).item():
```

The `dict.get(key, default)` function lets you get the value of a key from a dictionary if it exists, and if it doesn't exist, return a default value, in this case an empty dictionary.